PPAK_CVS/Sources/Scenes/Common/GoodsCell.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  GoodsCell.swift
+//  PPAK_CVS
+//
+//  Created by 김응철 on 2022/09/21.
+//
+import UIKit
+
+import Then
+import SnapKit
+
+final class GoodsCell: UICollectionViewCell {
+
+  // MARK: - properties
+  static let id = "GoodsCell"
+
+  private let containerView = UIView().then {
+    $0.layer.cornerRadius = 12.0
+    $0.layer.shadowOffset = CGSize(width: 0, height: 5)
+    $0.layer.shadowOpacity = 0.1
+    $0.layer.shadowColor = UIColor.black.cgColor
+    $0.backgroundColor = .white
+  }
+
+  private let logoView = UIView().then {
+    $0.layer.cornerRadius = 15.0
+    $0.backgroundColor = .blue
+  }
+
+  private let goodsLabel = UILabel().then {
+    $0.text = "코카)씨그랩피치탄산수350ML"
+    $0.font = .systemFont(ofSize: 16.0, weight: .bold)
+  }
+
+  private let priceLabel = UILabel().then {
+    $0.text = "1,500원"
+    $0.font = .systemFont(ofSize: 16.0, weight: .light)
+  }
+
+  private let descriptionLabel = UILabel().then {
+    $0.text = "(개당 750원)"
+    $0.font = .systemFont(ofSize: 14.0, weight: .light)
+  }
+
+  private let goodsImage = UIImageView().then {
+    $0.contentMode = .scaleAspectFit
+    $0.backgroundColor = .blue
+  }
+
+  private let eventView = UIView().then {
+    $0.layer.cornerRadius = 4.0
+    $0.backgroundColor = .blue
+  }
+
+  var stackView: UIStackView!
```

Comment:
기존의 StackView 메서드를 없애고, force-unwrapping을 제거한 코드입니다.
혹시, 따로 처리하신 이유가 있을까요?
```suggestion
  private lazy var stackView = UIStackView().then { stackView in
    let horiStack = UIStackView(arrangedSubviews: [priceLabel, descriptionLabel])
    horiStack.axis = .horizontal
    horiStack.spacing = 2.0
    [goodsLabel, horiStack].forEach {
      stackView.addArrangedSubview($0)
    }
      stackView.axis = .vertical
      stackView.spacing = 4.0
      stackView.alignment = .leading
  }
```